gateway/ln-gateway/src/lib.rs

```diff
@@ -503,6 +503,18 @@ impl Gateway {
                                         .await
                                         .expect("Failed to load gateway clients");
 
+                                        let scids = self
+                                            .scid_to_federation
+                                            .read()
+                                            .await
+                                            .keys()
+                                            .cloned()
+                                            .collect();
+                                        if let Err(e) = ln_client.update_fedimint_scids(scids).await {
+                                            warn!("Failed to update fedimint scids: {e:?}");
```

Comment:
Let's update this comment to mention that all HTLCs will be sent to `gatewayd` for inspection

Reply:
what comment? the warn error?

Reply:
Sorry, yes I meant the warn message. That might be useful for someone running the gateway to know

Reply:
The warning should only occur when running a new gateway+old extension, right?

Reply:
added